charts/datadog/values.yaml

```
@@ -503,10 +503,10 @@ datadog:
     enableOOMKill: false
 
     # datadog.systemProbe.enableRuntimeCompiler -- Enable the runtime compiler for eBPF probes
-    enableRuntimeCompiler: false
+    # enableRuntimeCompiler: false
```

I think you can do this:
```suggestion
    enableRuntimeCompiler:
```

---

But that might cause a change in your other template code. The benefit of having it defined, but unset, is that it shows up in documentation.

---

Maybe I'm wrong about the documentation though, I'm not 100% sure how that is processed.

---

Unfortunately this doesn't work for our use case - see my note in the README.